### ISSUE TITLE
Can't Set Notification Icon For LocalNotification

### ISSUE THREAD

**Issue by KSayed91** [ionic-team/capacitor#1966]
I tried to implement [Push Notification](https://capacitor.ionicframework.com/docs/apis/push-notifications) with Firebase messaging and [LocalNotification Plugin](https://capacitor.ionicframework.com/docs/apis/local-Notifications/). 
everything going well for Push Notification and the icon and color appear well as it in ApplicationManifset.xml

```
        <meta-data android:name="com.google.firebase.messaging.default_notification_icon" android:resource="@mipmap/notification_icon" />
        <meta-data android:name="com.google.firebase.messaging.default_notification_color" android:resource="@color/notification_color" />

```
![image](https://user-images.githubusercontent.com/5659755/64928101-43d18800-d81c-11e9-9334-9f7083f7aca2.png)

but when try to schedule local notification I got this strange icon without color 
![image](https://user-images.githubusercontent.com/5659755/64928084-d9b8e300-d81b-11e9-9cd1-7379c614ccd0.png)

Is there any way to configure icon for local notifications ?

I tried this code:

```
Plugins.LocalNotifications.schedule({
            notifications: [{
                ...
                smallIcon: 'res://notification_icon.png',
                ...
            }]
        });


```
but without any luck

**Comment by masimplo** [comment]
Take a look at this https://github.com/ionic-team/capacitor/pull/1830
You can specify it for all notifications at build time, not per notification on runtime.

**Comment by dl100463** [comment]
Not sure if it's the same problem but I'm also getting a default Android icon in the notification itself, no matter if the asset is specified for the notification at runtime, or all notifications in the capacitor.config.json at build time. The desired icon does however appear properly in the status bar at the top in both cases, just not inside the notification.

I also tried to change the plugin files themselves by setting the small icon there directly (which is supposed to impact both the status bar icon and notification icon according to the Android docs) through the NotificationCompat.Builder setSmallIcon (int icon) method, with assets in the right place, but that also didn't work somehow. Using the latest 1.2.1 version.

**Comment by jcesarmobile** [comment]
I can't reproduce.
Make sure you are using @capacitor/android 1.2.0 or newer and also don't use the image extension, just the name.
If you are still facing the issue, create a new issue providing a sample app and filling all the new template.

**Comment by Chandrasekar-G** [comment]
I'm facing the same issue.

**Comment by eryzerz** [comment]
> I'm facing the same issue.

This issue already solved on capacitor >= 1.3.0

If you don't want to update, just change the code within LocalNotification.java and core-plugin-definitions.d.ts manually based on this commit https://github.com/ionic-team/capacitor/pull/1830/commits/648d8a1e512c02fab2dfbf4dbf6df19e03c1c3df 

You can find LocalNotification.java in /node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/plugin/notification/LocalNotification.java and core-plugin-definitions.d.ts in /node_modules/@capacitor/core/dist/esm/core-plugin-definitions.d.ts
